app/templates/start-region-change.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base-' + display_region + '.html' %}
+
+{% from 'components/button/_macro.njk' import onsButton %}
+
+{% block main %}
+
+    <h1 class="u-mb-l u-fs-xxl">{{ _('Change of region') }}</h1>
```

Comment:
Why marked for language substitution? Is only ever displayed in English and following paragraph doesn't use internationalisation support?